libevm/ExtVMFace.h

```diff
@@ -271,13 +273,53 @@ class ExtVMFace: public evmc_context
     bytesConstRef data;       ///< Current input data.
     bytes code;               ///< Current code that is executing.
     h256 codeHash;            ///< SHA3 hash of the executing code
-    u256 salt;                ///< Values used in new address construction by CREATE2 
+    u256 salt;                ///< Values used in new address construction by CREATE2
     SubState sub;             ///< Sub-band VM state (suicides, refund counter, logs).
     unsigned depth = 0;       ///< Depth of the present call.
     bool isCreate = false;    ///< Is this a CREATE call?
     bool staticCall = false;  ///< Throw on state changing.
 };
 
+class EvmCHost : public evmc::Host
```

Comment:
I would better call it `EvmcHost`, but I mimicked the functions `toEvmC` / `fromEvmC` below

Reply:
This follows e.g. Java naming convention and it is fine. Following e.g. Go naming convention, this could be `EVMCHost`.